### ISSUE TITLE
Why doesn't it let me see the textdraw options

### ISSUE THREAD

**Issue by JoseEduardo10** [nexquery/samp-textdraw-editor#3]
I have a problem that I don't know why the menu appears invisible and I have tried everything and nothing has worked for me, any help?

**Comment by NIKEDev** [comment]
> I have a problem that I don't know why the menu appears invisible and I have tried everything and nothing has worked for me, any help?

True, same thing happning with me as i tried many time. when we click on the empty spaces function call and the dialogs occurs but we aren't able to see the icon.
see https://nike.has.rocks/tde.avi
